.gitmodules

```diff
@@ -4,5 +4,5 @@
 	branch = branch-22.11
 [submodule "morpheus_utils"]
 	path = external/utilities
-    url = https://github.com/nv-morpheus/utilities.git
+	url = ../utilities.git
```

Comment:
To make this work across forks, this needs to be `../../nv-morpheus/utilities.git`